apps/site-app/src/containers/MainNavigation/MainNavigationGreaterThanMobile/index.tsx

```diff
@@ -6,7 +6,7 @@ import { FormattedMessage } from 'react-intl';
 import HeaderLink from '@thecointech/site-base/components/HeaderLink';
 import { LanguageSwitcher} from '@thecointech/site-base/containers/LanguageSwitcher';
 
-import Logo from './logoAndName.svg';
+import Logo from './images/logoAndName.svg';
```

Comment:
This seems like a different style than what I've been doing in the past (normally I just drop images beside the component).

I have nothing against the new style - in fact, I can see it being a lot cleaner in a few instances.  If we are making a code-base wide stylistic change though, we should probably start a style guide?

Could you create a style-guide document in the docs folder, and add -just- this bit (images in subfolder)?  I'll make an effort to comform to the new guidelines, and fill out that doc with any further stylistic guidelines if/when making them in future.

Reply:
It's already pretty much done everywhere. I will create this doc in another PR.